samples/server/petstore/rust-server/output/rust-server-test/src/client/mod.rs

```diff
@@ -479,12 +503,19 @@ impl<F, C> Api<C> for Client<F> where
 
         let mut request = hyper::Request::new(hyper::Method::Post, uri);
 
+
+
+
+
```

Comment:
It looks as if there's lots of new extraneous newlines.

I'd be happier if they were removed, but I don't care particularly strongly.

Reply:
I'll leave them for now - there are future MRs in the works here.

In general, I think it's better for the mustache templates to be clean, than the code created by the generator - it's probably better if `cargo fmt` is just run over the result, rather than trying to get it perfectly clean.

That's especially true given we don't expect anyone to manually edit the generated files.

Reply:
I've pushed some changes to fix this - and get rid of most of the current extraneous new lines.